Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -84,6 +84,12 @@ YUI.add('ez-platformuiapp', function (Y) {
                 container: '.ez-confirmbox-container',
                 hideClass: 'is-confirmbox-hidden',
             },
+            languageSelectionBox: {
+                type: Y.eZ.LanguageSelectionBoxView,
+                service: Y.eZ.LanguageSelectionBoxViewService,
+                container: '.ez-languageselectionbox-container',
+                hideClass: 'is-languageselectionbox-hidden',
```

Comment:
you have to add this class to the `ez-platformui-app` element in shell.html.twig